drivers/usb/musb/Kconfig: remove the 'Glue Layer' choice.

In the config.omap configuration, we're setting CONFIG_USB_MUSB_OMAP2PLUS
to 'y' but after running the script to create our configuration file, it's
set to 'm', even if 'y' is valid. This result in an invalid configuration
and build failure (missing omap_musb_mailbox symbol in our kernel binary).

Removing the choice entry make things work.

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
Index: linux/drivers/usb/musb/Kconfig
===================================================================
--- linux.orig/drivers/usb/musb/Kconfig	2012-12-03 20:39:49.000000000 +0100
+++ linux/drivers/usb/musb/Kconfig	2012-12-05 15:06:20.000000000 +0100
@@ -33,9 +33,6 @@ config USB_MUSB_HDRC
 
 if USB_MUSB_HDRC
 
-choice
-	prompt "Platform Glue Layer"
-
 config USB_MUSB_DAVINCI
 	tristate "DaVinci"
 	depends on ARCH_DAVINCI_DMx
@@ -67,8 +64,6 @@ config USB_MUSB_UX500
 	tristate "U8500 and U5500"
 	depends on (ARCH_U8500 && AB8500_USB)
 
-endchoice
-
 choice
 	prompt 'MUSB DMA mode'
 	default USB_UX500_DMA if USB_MUSB_UX500
